### DISCUSSION TITLE
All Frequency Chat button

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5679] [Ideas and suggestions]
In addition to local and radio chat, there should be an all frequency radio chat. Radio channels are currently underutilized because people may forget which channel they are on, or are uncomfortable with switching between channels all the time. For instance, the captain could be switching between the mechanic channel, the engineering channel, and the sec channel, and those people could be in turn swapping between channels to network. This leads to the radio channels being unworkable and therefore unused. This could be rectified with a way to speak on all channels at once. Perhaps the "C" key by default could be made an all frequency chat button.

**Comment by James-E-A** [comment]
Alternative suggestion to fill this use-case: #6147

> receive _and_ transmit on a primary channel; receive-only on a second channel (perhaps split to left and right ear for each channel / a bit like a guard channel on military aircraft).
